File: OSCommon/org/optimizationservices/oscommon/datastructure/osresult/ResultHeader.cs
using System;

namespace org.optimizationservices.oscommon.datastructure.osresult{
	/// <summary>
	/// A data structure class that corresponds to an xml element in the OSrL schema. 
	/// @author Jun Ma	
	/// @version 1.0, 09/01/2005
	/// @since OS 1.0
	/// @copyright (c) 2005
	/// </summary>
	public class ResultHeader{
		
		public GeneralStatus generalStatus = new GeneralStatus();

		public string serviceURI = "";
		
		public string serviceName = "";

		public string instanceName = "";

		public string jobID = "";

		public DateTime time = new DateTime(1970, 1, 1, 0, 0, 0);

		public string message = "";

	}//class ResultHeader
}//namespace

File: OSCommon/bak/osresult_complex/ResultHeader.java
/**
 * @(#)ResultHeader 1.0 03/14/2004
 *
 * Copyright (c) 2004
 */
package org.optimizationservices.oscommon.datastructure.osresult;

import java.util.GregorianCalendar;

/**
*
* <p>A data structure class that corresponds to an xml element in the OSrL schema.  
*  
* </p>
* @author Robert Fourer, Jun Ma, Kipp Martin
* @version 1.0, 03/14/2004
* @since OS 1.0
*/
public class ResultHeader {

	
	public GeneralStatus generalStatus = new GeneralStatus();

	public String serviceURI = "";
	
	public String serviceName = "";

	public String instanceName = "";

	public String jobID = "";

	public GregorianCalendar time = null;

	public String message = "";

}//ResultHeader
